program/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "address-lookup-table"
+version = "0.1.0"
+edition = "2021"
+readme = "./README.md"
+license-file = "../LICENSE"
+publish = false
+
+[package.metadata.solana]
+program-id = "AddressLookupTab1e1111111111111111111111111"
+
+[features]
+no-entrypoint = []
```

Comment:
Maybe this is a good time to figure out how to identify and document certain standards for all of these programs? This could include bringing habits over from SPL, as well as addressing longstanding issues like this one.

cc @lorisleiva 

In that spirit, happy to address this problem, and others, but let's make sure we remember to do it on all programs and have somewhere to reference these standards.

Reply:
For existing programs, it'll be much trickier, since people are already importing those crates with `no-entrypoint`. Since these are all new, we have a bit more flexibility, so I would go with doing the right thing.

Reply:
Ohh yeah I didn't mean the existing ones, just the reimplementations.

Reply:
I've added a commit for this. Do we even need a feature? I was able to just annotate the entrypoint module like so:

```rust
#[cfg(target_os = "solana")]
mod entrypoint;
```

Reply:
Sorry, we do need the feature -- if a BPF program imports this crate, then it will have two entrypoints defined and fail compilation.

Reply:
Ahh okay, good shout. I've updated the commit. We just have to remember to use `--features bpf-entrypoint` whenever we call `cargo test-sbf`.

Reply:
Ah good point... that should be ok, but I'll have to think about it. How do you feel about that?

Reply:
I think it's fine, but we should consider adding it to the platform tools.

Reply:
https://github.com/solana-program/.github/issues/15